app/views.py

```diff
@@ -22,416 +22,515 @@
+            search_start_date=search_start_date,
+            search_end_date=search_end_date,
+            search_artist=search_artist)
+        if search_artist == "%":
+            search_artist = ""
+        return jsonify(data)
     else:
-      search_start_date = "1969-01-01"
-      search_end_date = "3000-01-01"
-
-    #search artist
-    search_artist = request.args.get("search_artist", "%")
-
-    if search_artist == "":
-        search_artist = "%"
-
-    video_scope = request.args.get("video_scope")
-
-    data = viewsModel.getvideodata(user_id = session['session_user_id'], video_scope = video_scope, search_start_date = search_start_date, search_end_date = search_end_date, search_artist = search_artist)
-    if search_artist == "%":
-        search_artist = ""
-    return jsonify(data)
-  else:
-    return redirect(url_for('login'))
-  return "success"
+        return redirect(url_for('login'))
+    return "success"
 
 
 @app.route('/login')
 def login():
     return google.authorize(callback=url_for('authorized', _external=True))
 
+
 @app.route('/logout')
 def revoke_token():
-  if 'google_token' in session:
-    res = google.get('https://accounts.google.com/o/oauth2/revoke', data={'token': session['google_token'][0]})
-    session.pop('user_email', None)
-    session.pop('google_token', None)
-    return redirect('/')
-  return redirect(url_for('login'))
+    if 'google_token' in session:
+        res = google.get('https://accounts.google.com/o/oauth2/revoke',
+                         data={'token': session['google_token'][0]})
+        session.pop('user_email', None)
+        session.pop('google_token', None)
+        return redirect('/')
+    return redirect(url_for('login'))
+
 
 @app.route('/oauth2callback')
 @google.authorized_handler
 def authorized(resp):
     if resp is None:
         return 'Access denied: reason=%s error=%s' % (
-            request.args['error'],
-            request.args['error_description']
-        )
+            request.args['error'], request.args['error_description'])
     session['google_token'] = (resp['access_token'], resp['id_token'])
     res = google.get('https://www.googleapis.com/plus/v1/people/me')
     google_object = res.data
-    if (len(google_object['emails']) > 0):
-      session['user_email'] = (google_object['emails'][0]['value'])
-      sql_session.rollback()
-      email_in_db = sql_session.query(models.User).filter_by(email=session['user_email']).first()
-      last_id_query = sql_session.query(func.max(models.User.id))
-      last_id = last_id_query.one()
-
-      if not email_in_db:
-        new_user = models.User(id=last_id[0] + 1,
-                              verification_level=100,
-                              email=session['user_email'])
-        sql_session.add(new_user)
-        sql_session.commit()
-        session['session_user_id'] = last_id[0] + 1
-      else:
-        session['session_user_id'] = email_in_db.id
+    if len(google_object['emails']) > 0:
```

Comment:
One style issue not caught by an autoformatter is the unnecessary use of parentheses in an `if` statement. There's only one condition here, so no need to wrap it in parentheses.